### PR TITLE
number op range to i64 as eip-1985

### PR DESCRIPTION
**Pull request by bchyl**

### CODE REVIEW

File: src/zkevm_specs/evm/typing.py [hunk 1]
```diff
@@ -27,7 +27,10 @@ class Block:
     gas_limit: U64
 
     # For other fields, we follow the size defined in yellow paper for now.
-    number: U256
+    # as described in https://eips.ethereum.org/EIPS/eip-1985,
+    # block number, timestamp is a range between 0 and 0x7fffffffffffffff
```
Comment (CPerezz) [inline src/zkevm_specs/evm/typing.py:31]:
```suggestion
    # block number and timestamp are inside a range between 0 and 0x7fffffffffffffff
```

**Review by CPerezz — APPROVED** [review]
LGTM modulo the nit! :)

File: src/zkevm_specs/evm/typing.py (hunk 1 above)
Comment (bchyl) [reply src/zkevm_specs/evm/typing.py]: got it

**Comment by CPerezz** [comment]
@bchyl can we resolve conflicts? Once done we're good to merge I think.
Maybe @ed255 or @ChihChengLiang can take a quick look before.

**Review by ed255 — APPROVED** [review]
LGTM!  Thanks for updating the spec :)

**Comment by bchyl** [comment]
> 



> @bchyl can we resolve conflicts? Once done we're good to merge I think. Maybe @ed255 or @ChihChengLiang can take a quick look before.

done

**Pull request merged**